### PR TITLE
Added check in order to push only visits to queue

### PR DESCRIPTION
**Pull request by oalexandru**

### CODE REVIEW

**Comment by arupdvsa** [comment]
with this change the existing unit test for retroGenInit for context("when fetching an activity stream") should have failed but there isn't a scenario for activityType non-visit 
1) To add scenario activityType non-visit and adjust the existing test accordingly
2) To add context for non-visit type and test no event .. also refer the Jira-defect id for reference documentation 
3) JSON.parse(fs.readFileSync(path.resolve(__dirname, "../resources/stream-event.json"), "utf8")) can be moved to beforeEach

**Comment by oalexandru** [comment]
> with this change the existing unit test for retroGenInit for context("when fetching an activity stream") should have failed but there isn't a scenario for activityType non-visit
> 
> 1. To add scenario activityType non-visit and adjust the existing test accordingly
> 2. To add context for non-visit type and test no event .. also refer the Jira-defect id for reference documentation
> 3. JSON.parse(fs.readFileSync(path.resolve(__dirname, "../resources/stream-event.json"), "utf8")) can be moved to beforeEach


1.&2. Added non visit items in testData so that "when fetching an activity stream" unit test covers now also the filtering of wait times. 
3. As discussed I see no reason for moving this into a beforeEach statement in this case

File: src/services/StreamService.ts [hunk 1]
```diff
@@ -8,9 +8,9 @@ class StreamService {
      * to a JS object
      * @param event
      */
-    public static getActivitiesStream(event: any) {
-        return event.Records.filter((record: DynamoDBRecord) => { // Retrieve "MODIFY" events
-            return record.eventName === "MODIFY";
+    public static getVisitsStream(event: any) {
```
Comment (arupdvsa) [inline src/services/StreamService.ts]: going forward tslint will error `any` . Can you derive a type from usage if quick and easy ... if not then please leave a comment for todo .. so this is picked as part of the typescript migration task

File: src/services/StreamService.ts [hunk 2]
```diff
@@ -8,9 +8,9 @@ class StreamService {
...
+        return event.Records.filter((record: DynamoDBRecord) => { // Retrieve "MODIFY" visit events
+            return record.eventName === "MODIFY" && record.dynamodb && record.dynamodb.NewImage && record.dynamodb.NewImage.activityType.S === "visit";
```
Comment (KHodderDVSA) [inline src/services/StreamService.ts]: I really hoped to find a good "optional chaining" option for TS, but apparently not. https://medium.com/openmindonline/js-monday-10-optional-chaining-proposal-d9c42b5da517

File: src/services/StreamService.ts (hunk 1 above)
Comment (KHodderDVSA) [reply src/services/StreamService.ts]: This is a production bug, so I'd suggest the "todo" approach for now. We'll fix it up when it's not affecting prod.

File: src/services/StreamService.ts (hunk 2 above)
Comment (arupdvsa) [reply src/services/StreamService.ts]: Yeah the remaining proposed changed could be picked during the typescript migration

**Pull request merged**